project/make/gccgo

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+BINARY_NAME="docker-$VERSION"
+BINARY_EXTENSION="$(binary_extension)"
+BINARY_FULLNAME="$BINARY_NAME$BINARY_EXTENSION"
+
+source "$(dirname "$BASH_SOURCE")/.go-autogen"
+
+if [ "$IAMSTATIC" == "true" ]; then 
+	EXTLDFLAGS_STATIC_DOCKER="-static -Wl,--no-export-dynamic $(echo $EXTLDFLAGS_STATIC_DOCKER | sed "s/$EXTLDFLAGS_STATIC//")"
+else
+	EXTLDFLAGS_STATIC_DOCKER="-static-libgo $(echo $EXTLDFLAGS_STATIC_DOCKER | sed "s/$EXTLDFLAGS_STATIC//")"
+fi
```

Comment:
This whole `if` statement concerns me.  The whole reason for the subshell and environment variable resetting in `dynbinary`/`dyngccgo` is to remove the "static flags" so that `binary` creates a dynamic executable instead, and I'd really prefer to see this file work the same way for consistency.

For example, I don't see any reason why we need to do anything static for the "dynamic binary" case, including `-static-libgo` -- after all, getting dynamic includes is the whole point of having a separate static `dockerinit` binary.